Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -4309,7 +4309,7 @@ MediaSourceEnabled:
 
 MediaSourceInlinePaintingEnabled:
   type: bool
-  status: unstable
+  status: stable
```

Comment:
@jernoble: Could you confirm that this status change is accurate? It reflects that the feature is always enabled on Mac.

Reply:
I think it’s risky to have a single status express the state of the feature, and have a road map to enforcing the relationship, when the status may be different per platform and even per OS version. The fact that `defaultValue` is conditional here is quite important!

Reply:
Indeed. Currently, macOS is the platform that the statuses are aligned to, because Mac Safari is the only browser that exposes feature status to web developers.

IMO, the next step will be making it possible for a feature's status to be conditional, so that they can accurately describe behavior on different platforms.